.github/workflows/tests.yml

```diff
@@ -13,18 +13,47 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ['2.7', 'pypy-2.7', '3.9', '3.10', '3.11']
+        python-version: ['pypy-2.7', '3.9', '3.10', '3.11']
         exclude:
           - os: macos-latest
             python-version: 'pypy-2.7'
+          - os: windows-latest
+            python-version: 'pypy-2.7'
     steps:
       - uses: actions/checkout@v2
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Show Python Version
+        run: |
+          python --version
+      - name: Install dependencies
+        run: |
+          python -m pip install wheel
+      - name: Install
+        run: |
+          python -m pip -v install --global-option=--require-scandir-extension .
       - name: Run tests
+        run: |
+          python test/run_tests.py
+  build27:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    container:
+      image: python:2.7.18-buster
+    steps:
+      - uses: actions/checkout@v2
+      - name: Show Python Version
```

Comment:
Let's remove the "Show Python Version" step here too, as it's explicit in the container image name.